pypanther/main.py

```diff
@@ -148,14 +148,6 @@ def setup_parser() -> argparse.ArgumentParser:
     )
     setup_list_log_types_parser(list_log_types_parser)
 
-    # convert command
-    convert_parser = subparsers.add_parser(
-        name="convert",
-        help="Convert Panther Analysis rules",
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
-    )
-    setup_convert_parser(convert_parser)
```

Comment:
Make sure to remove `setup_convert_parser` from the imports as well.

Reply:
Removed!